test/jdk/java/io/ObjectStreamClass/TestOSCClassLoaderLeak.java

```diff
@@ -61,8 +61,7 @@ public void testClassLoaderLeak() throws Exception {
         con = null;
         assertNotNull(myOwnClassLoaderWeakReference.get());
 
-        ForceGC gc = new ForceGC();
-        assertTrue(gc.await(() -> myOwnClassLoaderWeakReference.get() == null));
+        assertTrue(ForceGC.wait(() -> myOwnClassLoaderWeakReference.get() == null));
```

Comment:
The call to ref.get() can create a strong reference to the object; depending on the timing it might interfere with the GC in process.
`ref.refersTo(null)` is preferred over `ref.get() == null` 

Here and all subsequent changes.

Reply:
Good point.  I learned a lot these days from PRs comments!  Thank you!

The patch is changed to use refersTo().  I will check if there are somewhere else that use` ref.get() == null` improperly.